less/_toolbar.less

```diff
@@ -125,23 +125,14 @@
 		}
 	}
 	.folder {
-		.txt-link {
+		&.not-empty {
 			.icon {
-				position: absolute;
-				left: 2px;
-
-				width: 27px;
-				height: 15px;
-
-				background-color: @usernav-item-folder-bgcolor;
-				background-image: url(@usernav-item-folder-icon);
-				background-position: 8px -1px;
-				background-repeat: no-repeat;
+				background-position: 0 -101px;
```

Comment:
Actually, the `.not-empty` class should also be moved because that is specific to the sprite.